doc/api/tls.md

```diff
@@ -621,7 +623,7 @@ For example:
 ```
 
 If the peer does not provide a certificate, `null` or an empty object will be
-returned.
+returned. XXX uh, which one is it?
```

Comment:
Just marking this so this doesn't accidentally get merged without this being resolved.